packages/react-components/theme-designer/.npmignore

```diff
@@ -16,7 +17,7 @@ __tests__
 *.api.json
 *.log
 *.spec.*
-*.stories.*
+*.cy.*
```

Comment:
This looks strange

Reply:
looks like nobody migrated this package to new structure yet. I'll undo those. thanks for eagles eye check!